migration_original/ODS1Stage/tables/Base/ProviderTraining/spu_translated_ProviderTraining.sql

```diff
@@ -80,8 +87,7 @@ insert_statement := ' insert
 
 merge_statement := ' merge into base.providertraining as target using 
                    ('||select_statement||') as source 
-                   on source.providerid = target.providerid
-                   WHEN MATCHED then delete
+                   on source.providerid = target.providerid and source.trainingid = target.trainingid
```

Comment:
please add the update clause for when matched

Reply:
Done